changelog/55295.changed

```diff
@@ -0,0 +1 @@
+renamed `keep_jobs`, specifying job cache TTL in hours, to `keep_jobs_seconds`, specifying TTL in seconds
```

Comment:
Just a small request. Can you change the filename to `.deprecated`. We have not changed anything yet, because you can still use `keep_jobs` so the describption should reflect that `keep_jobs` is on a deprecation path and will be removed in favor of keep_jobs_seconds in the Chlorine release.